content/docs/user-guide/visualizing-plots.md

```diff
@@ -279,6 +279,9 @@ DVC has the following built-in plot templates:
   [example](/doc/command-reference/plots/show#example-confusion-matrix)
 - `confusion_normalized` - confusion matrix with values normalized to <0, 1>
   range
+- `bar_horizontal` - horizontal bar plot, see
+  [example](/doc/command-reference/plots/show#example-horizontal-bar-plot)
+- `bar_horizontal_sorted` - horizontal bar plot sorted by bar size
```

Comment:
Minor: I think "bar chart" is much more widely used. But surprisingly (for me) term "bar plot" is [not insignificant](https://trends.google.com/trends/explore?q=bar%20plot,bar%20chart) out there 🤷🏼